src/clients/googleAnalytics.ts

```diff
@@ -7,6 +7,16 @@ export type EventMap = {
   mint_vai: { amount: string };
   repay_vai: { amount: string };
   collateral_toggle: { status: 'enter' | 'exit' | 'in_repayment'; market: string };
+  create_proposal: {
+    status:
+      | 'show_modal'
+      | 'proposal_in_progress'
+      | 'submit'
+      | 'submitted_with_invalid_parameters'
+      | 'proposal_submission_failed';
+  };
+  delegate_voting: { type: 'delegate' | 'manual' };
+  collect_rewards: { tokens: string };
```

Comment:
How about making that name a bit more explicit? `tokenAddresses`

Reply:
Yeah, that makes sense. I imagine this will change though because the token name is probably more useful but that isn't available here from the current data model

Reply:
Got you. When that happens then we can rename it to `tokenNames` :)